cmd/proxy/main.go

```diff
@@ -345,17 +344,19 @@ func createVersionManagerTestOrProduction(
 			AddressPubkeyConverter: cfg.AddressPubkeyConverter,
 		}
 
-		return createVersionManager(testCfg, ecCfg, exCfg, ctx.GlobalString(walletKeyPemFile.Name))
+		return createVersionManager(testCfg, ecCfg, exCfg, ctx.GlobalString(walletKeyPemFile.Name), false)
 	}
 
-	return createVersionManager(cfg, ecCfg, exCfg, ctx.GlobalString(walletKeyPemFile.Name))
+	isRosettaOn := ctx.GlobalBool(startAsRosetta.Name)
+	return createVersionManager(cfg, ecCfg, exCfg, ctx.GlobalString(walletKeyPemFile.Name), isRosettaOn)
 }
 
 func createVersionManager(
 	cfg *config.Config,
 	ecConf *erdConfig.EconomicsConfig,
 	exCfg *erdConfig.ExternalConfig,
 	pemFileLocation string,
+	isRosettaOn bool,
```

Comment:
isRosettaEnabled (On can mean something else :D )

Reply:
renamed to `isRosettaModeEnabled`